rfcs/0145-doc-strings.md

```diff
@@ -134,16 +130,16 @@ Each subsection here contains a decision along with arguments and counter-argume
 <summary>Arguments</summary>
 
 - (+) Stays primarily compatible with the currently used comments in lib.
-    - (-) Outside the ./lib folder, nixodc is not a convention. 
+    - (-) Outside the ./lib folder, nixodc is not a convention.
 - (-) Is not widely understood.
-- (-) Is not widely used. (e.g., Outside the /lib folder)
 - (-) The format is specific, should be more generic, and follow established conventions. (e.g., Markdown, Doxygen, JSdoc, Haddock)
 - (-) The format is too complex and needs a whole list of complex rules to describe the exact format and content rules; this can also be observed in many false-formatted comments that do not render correctly.
     - (-) The sections are implicitly continued.
     - (-) The predefined section headers/keywords are very close to natural language and may accidentally be used.
         - (-) Syntax to start new sections does not follow conventions (e.g., `Example:`) In contrast JSdoc defines `@Example`; Markdown `# Example`; Doxygen `@example`; Haskell-Haddock `>>>`
     - (-) The content is implicitly rendered differently depending on predefined rules. Those rules need to be understood/feel natural.
-- (+) It Is already quite established and has some adoptions outside the `/lib` folder.
+- (+) Is already quite established in /lib.
```

Comment:
```suggestion
```

What is the difference between first and this bullet points? Or what different aspects they try to capture?

```markdown
- (+) Stays primarily compatible with the currently used comments in lib.
```
```markdown
- (+) Is already quite established in /lib.
```